### PR TITLE
Updated the instructions on how to run the app

### PR DESCRIPTION
**Pull request by otzy007**
In order to avoid the confusion when the database structure is created I've updated the run instructions.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://first-start-instructions.foundedx-founded-in-x-6b44a3bf69bce8a4cd3a.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://first-start-instructions.foundedx-founded-in-x-6b44a3bf69bce8a4cd3a.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: README.md [hunk 1]
```diff
@@ -91,11 +91,16 @@
   2. Change config.py with the variables suitable for you and add the file to .gitignore
 
 ## Access the app
+First time run the following command to create the database and start the app:
```
Comment (dbsima) [inline README.md:94]: It doesn't start the app. Maybe it's a bad practice.

File: README.md [hunk 2]
```diff
@@ -91,11 +91,16 @@
...
 ```shell
 python run.py --setup
 ```
 Go to [http://localhost:5000](http://localhost:5000) for website and [http://localhost:5000/admin](http://localhost:5000/admin) for the administrator interface
 
+Later you can start the app with the command:
```
Comment (dbsima) [inline README.md:100]: After you run python run.py --setup, it says to run without "--setup"

**Comment by otzy007** [comment]
oops. Sorry :))

**Pull request closed without merging**